### PR TITLE
Add details about vxlan tunnel

### LINKED ISSUE #450: Update docs with vxlan cable driver support
What would you like to be added:
vxlan cable driver details needs to be added in docs
Why is this needed:
As a part of #674 a new vxlan cable driver will be supported. The docs needs to be updated with the appropriate details like configuration, and trouble shooting.

### PR DESCRIPTION
**Pull request by aswinsuryan**
Fixes: #450

Signed-off-by: Aswin Surayanarayanan <asuryana@redhat.com>

### CODE REVIEW

File: src/content/getting-started/architecture/gateway-engine/_index.en.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ that maintains the tunnels. The following implementations are available:
 
 * an IPsec implementation using [Libreswan](https://libreswan.org/). This is currently the default.
 * an implementation for [WireGuard](https://www.wireguard.com/) (via the [wgctrl](https://github.com/WireGuard/wgctrl-go) library).
+* an un-encrypted tunnel implementation using vxlan.
```
Comment (nyechiel) [inline src/content/getting-started/architecture/gateway-engine/_index.en.md]:
```suggestion
* an un-encrypted tunnel implementation using VXLAN.
```

File: src/content/operations/deployment/helm/_index.en.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ helm install submariner-operator submariner-latest/submariner-operator \
         --set broker.token="${SUBMARINER_BROKER_TOKEN}" \
         --set broker.namespace="${BROKER_NS}" \
         --set broker.ca="${SUBMARINER_BROKER_CA}" \
-        --set submariner.cableDriver=libreswan \ # or wireguard
+        --set submariner.cableDriver=libreswan \ # wireguard or vxlan
```
Comment (nyechiel) [inline src/content/operations/deployment/helm/_index.en.md]:
```suggestion
        --set submariner.cableDriver=libreswan \ # or wireguard or vxlan
```

File: src/content/operations/deployment/subctl/_index.en.md [hunk 3]
```diff
@@ -93,7 +93,7 @@ deployment.
 <!-- markdownlint-disable line-length -->
 | Flag                               | Description
 |:-----------------------------------|:----------------------------------------------------------------------------|
-| `--cable-driver` `<string>`        | Cable driver implementation. Available options are `libreswan` (default) and `wireguard`
+| `--cable-driver` `<string>`        | Cable driver implementation. Available options are `libreswan` (default) `wireguard` and 'vxlan'
```
Comment (nyechiel) [inline src/content/operations/deployment/subctl/_index.en.md]:
```suggestion
| `--cable-driver` `<string>`        | Cable driver implementation. Available options are `libreswan` (default), `wireguard` and 'vxlan'
```

File: src/content/operations/nat-traversal/_index.en.md [hunk 4]
```diff
@@ -94,7 +94,7 @@ port of packets, but **this will happen as soon as two connections collide
 over the same source port**.
 {{% /notice %}}
 
-## UDP Dataplane Protocol (IPsec & WireGuard)
+## UDP Dataplane Protocol (IPsec WireGuard & Vxlan)
```
Comment (nyechiel) [inline src/content/operations/nat-traversal/_index.en.md]:
```suggestion
## UDP Dataplane Protocol (IPsec, WireGuard or VXLAN)
```

**Review by nyechiel — COMMENTED** [review]
A few nits. Thanks @aswinsuryan

**Review by nyechiel — APPROVED** [review]

**Review by skitt — APPROVED** [review]

**Pull request merged**